code/style.css

```diff
@@ -152,29 +178,42 @@ button {
   justify-content: center;
   background: var(--secondary);
   position: absolute;
-  top: 0;
-  bottom: 0;
+  /* top: 0;
+  bottom: 0; */
   left: 0;
   right: 0;
-  height: 100vh;
+  /* height: 100vh; */
 }
 
 .win-or-lose {
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  max-width: 700px;
+  max-width: 1000px;
   text-align: center;
+  margin-top: 10px;
+  margin-bottom: 10px;
 }
 
-.win-or-lose img {
+.guess-who-icon {
   align-self: flex-start;
   margin-left: 15%;
 }
 
-.win-or-lose h1 {
-  margin-bottom: 32px;
+/* .win-or-lose h1 {
+  margin-bottom: 32px; 
+} */
+
+.win-or-lose-text {
+  font-size: 20px;
+}
+
+.winner-img {
+  height: 50%;
+  width: 50%;
+  /* object-fit: contain; */
+  border: black solid 2px;
 }
 
 @media (max-width: 768px) {
```

Comment:
Maybe you could add some more styling to the media queries section in your code - to also allow the text "Guess on [name]" to be shown on the card in the mobile view. Also, in the mobile view there is a white section added to the bottom of the page that I'm not sure is intended to be there?